oci.go

```diff
@@ -34,13 +33,13 @@ const (
 	cgroupsMountType = "cgroup"
 )
 
-var (
-	errNeedLinuxResource = errors.New("Linux resource cannot be empty")
-)
-
 var cgroupsMemDirPath = "/sys/fs/cgroup"
 
 func containerExists(containerID string) (bool, error) {
+	if containerID == "" {
```

Comment:
Argh. Got that wrong.

That said, we're not checking at several places that containerID is not "". One, at the start, should be enough, not at the start + in containerExists